### DISCUSSION TITLE
Time based notification and selectable username only authentication.

### DISCUSSION THREAD

**Discussion by berrystephenw** [berrystephenw/OctoText#96] [General]
This is a test release that allows for notifications that are time based instead of percentage of file completed. Still more work to be done on this feature but if anyone is interested in testing it that would be appreciated.

Another feature was added for SMTP authentication for servers that require only a username (not an email address) for logins. 

<hr /><em>This discussion was created from the release <a href='https://github.com/berrystephenw/OctoText/releases/tag/0.3.2rc1'>Time based notification and selectable username only authentication.</a>.</em>

**Comment by antoni-g** [comment]
is this time-based only while printing, or time-based in general? (ie, if no print is happening)

**Reply by berrystephenw** [reply]
Only while printing.
